stock_picking_auto_create_lot_serial_only/views/product_template.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2020 ACSONE SA/NV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_template_property_form" model="ir.ui.view">
+        <field
+            name="name"
+        >product.template.form (in stock_picking_type_auto_create_serial_only)</field>
+        <field name="model">product.template</field>
+        <field
+            name="inherit_id"
+            ref="stock_picking_auto_create_lot.view_template_property_form"
+        />
+        <field name="arch" type="xml">
+            <field name="tracking" position="attributes">
+                <attribute
+                    name="attrs"
+                >{'invisible':[('tracking', '!=', 'serial')]}</attribute>
+            </field>
```

Comment:
By default tracking field is none... so the field in view is invisible... How do I can change this value if the field is invisible.
![Selección_029](https://user-images.githubusercontent.com/7689807/98511748-81397600-2265-11eb-86e1-b1d5582a2e65.png)